app-games/src/main/java/com/aptoide/android/aptoidegames/AptoideAsyncImage.kt

```diff
@@ -73,6 +95,102 @@ fun AptoideAsyncImage(
   )
 }
 
+@Composable
+fun AptoideAsyncImageWithFullscreen(
+  modifier: Modifier = Modifier,
+  data: Any?,
+  placeholder: Boolean = true,
+  contentDescription: String?,
+  transformations: Transformation? = null,
+  colorFilter: ColorFilter? = null,
+  screenshots: List<String?> = emptyList(),
```

Comment:
Change list type to Any? to keep consistency, so that the composable can be called with other types of data. Also change the variable name, since it might not only be screenshots